wgpu/enums.py

```diff
@@ -409,3 +411,11 @@ class ErrorFilter(Enum):
     validation = "validation"
     out_of_memory = "out-of-memory"
     internal = "internal"
+
+
+class PipelineStatisticName(Enum):  # wgpu native
+    VertexShaderInvocations = "vertex-shader-invocations"
+    ClipperInvocations = "clipper-invocations"
+    ClipperPrimitivesOut = "clipper-primitives-out"
+    FragmentShaderInvocations = "fragment-shader-invocations"
+    ComputeShaderInvocations = "compute-shader-invocations"
```

Comment:
We keep the main API free of backend-specific stuff like this. So we can do one of two things:

* Add an enum to `extras.py` (and import it in the `wgpu.backends.wgpu_native` namespace like the rest.
* Make it an "implicit enum" where we simply document for the `statistics` argument what the possible values are.

The changes to the code that generates `_mappings.py` is fine; we need to be able to convert these strings to ints.